cmake/OpenCVFindProtobuf.cmake

```diff
@@ -89,3 +123,8 @@ if(HAVE_PROTOBUF)
     BUILD_PROTOBUF THEN "build (${Protobuf_VERSION})"
     ELSE "${__location} (${Protobuf_VERSION})")
 endif()
+
+if(HAVE_ABSL_STRINGS AND HAVE_ABSL_LOG)
+  list(APPEND CUSTOM_STATUS absl)
+  list(APPEND CUSTOM_STATUS_absl "      abseil-cpp:" "YES (${ABSL_STRINGS_VERSION})" )
```

Comment:
`abseil-cpp` is a direct dependency of Protobuf, not OpenCV. Not sure that it has to be reported.

Reply:
When using protobuf v22 or later, 
the dnn module directly references `abseil-cpp` library ( via *.pb.cc converted *.pb with `protoc` ).

Unfortunally, there is also a possibility that OpenCV Users may use a combination of `protobuf` and `abseil-cpp` versions that are not expected.

I think it is better to leave abseil-cpp version information as a cmake log.
It is helpful to investifate compile/runtime errors

```bash
kmtr@kmtr-VMware-Virtual-Platform:~/work/build4-main_cxx17/modules/dnn/CMakeFiles/opencv_dnn.dir$ nm attr_value.pb.cc.o | c++filt | grep "absl" | head -10
                 U absl::lts_20230125::log_internal::LogMessage::OstreamView::stream()
                 U absl::lts_20230125::log_internal::LogMessage::OstreamView::OstreamView(absl::lts_20230125::log_internal::LogMessage::LogMessageData&)
                 U absl::lts_20230125::log_internal::LogMessage::OstreamView::~OstreamView()
                 U absl::lts_20230125::log_internal::LogMessage::CopyToEncodedBuffer(std::basic_string_view<char, std::char_traits<char> >, absl::lts_20230125::log_internal::LogMessage::StringType)
0000000000000000 W absl::lts_20230125::log_internal::LogMessage& absl::lts_20230125::log_internal::LogMessage::operator<< <14>(char const (&) [14])
0000000000000000 W absl::lts_20230125::log_internal::LogMessage& absl::lts_20230125::log_internal::LogMessage::operator<< <22>(char const (&) [22])
0000000000000000 W absl::lts_20230125::log_internal::LogMessage& absl::lts_20230125::log_internal::LogMessage::operator<< <23>(char const (&) [23])
0000000000000000 W absl::lts_20230125::log_internal::LogMessage& absl::lts_20230125::log_internal::LogMessage::operator<< <2>(char const (&) [2])
0000000000000000 W absl::lts_20230125::log_internal::LogMessage& absl::lts_20230125::log_internal::LogMessage::operator<< <34>(char const (&) [34])
0000000000000000 t absl::lts_20230125::log_internal::LogMessage& absl::lts_20230125::log_internal::LogMessage::operator<< <48>(char const (&) [48]) [clone .isra.0]
```

Reply:
I thought about it additionally.

When a user builds a protobuf, it is fully expected that the appropriate abseil-cpp version will be applied in the git submodule.

Therefore, I would like to delete the version log. thank you very much!

e.g.  https://github.com/protocolbuffers/protobuf/tree/main/third_party

> abseil-cpp @ fb3621f  Update to latest absl LTS patch 20230802.1 (#14145)